### ISSUE TITLE
Additional fields on registration form are not saved

### ISSUE THREAD

**Issue by kryten87** [stormpath/stormpath-laravel#25]
This may or may not be a bug - it's not clear what the expected behaviour is.

I added a field to my registration form for a company name, like so:

```
        "form" => [
            "fields" => [
                ...
                "company" => [
                    "enabled" => true,
                    "name" => "company",
                    "placeholder" => "Company",
                    "required" => true,
                    "type" => "text"
                ],
                ...
```

The field is visible on the form & validation works great. However, when the user is registered, this value is ignored.

I expected this value to be saved as a custom value on the `Account` object, but it's not. If something else is supposed to happen, please let me know!

**Comment by bretterer** [comment]
@Kryten0807 Thank you for your input here.  This is currently working as planned and not adding these fields.  I a very near future release, this will be available.

-Brian

**Comment by kryten87** [comment]
Awesome! Thanks for letting me know.

**Comment by bretterer** [comment]
Merged into Version 0.2.0  Thank you for the PR!
